src/pages/profile/Profile.tsx

```diff
@@ -1,8 +1,25 @@
+import * as S from './Profile.style';
+
 const Profile = () => {
   return (
-    <div>
-      <div>profile</div>
-    </div>
+    <>
+      <S.ProfileWrapper>
+        <S.Profile>
+          <img src='/img/ProfileImg.png' alt='' />
+          <S.ProfileDescription>
+            <S.Name>이무성</S.Name>
+            <S.Description>저는 5년차 개발자입니다. 주로 react 개발을 위주로 하는 중 입니다.</S.Description>
+          </S.ProfileDescription>
+        </S.Profile>
+        <S.MentorSkills>
+          <S.Mentor>멘토</S.Mentor>
+          <S.Skills>Skills</S.Skills>
+        </S.MentorSkills>
+      </S.ProfileWrapper>
+      <S.ProfileBtn>All activity</S.ProfileBtn>
+      <S.ProfileBtn>Article</S.ProfileBtn>
+      <S.ProfileBtn>Comment</S.ProfileBtn>
+    </>
```

Comment:
view단이 너무 붙어있어서 흐름을 파악하기가 힘든 것 같아요.
어느 정도 의미가 나뉘는 곳에서는 띄어쓰는게 좋다는 생각이 듭니다.

```suggestion
      <S.ProfileWrapper>
        <S.Profile>
          <img src='/img/ProfileImg.png' alt='' />
          
          <S.ProfileDescription>
            <S.Name>이무성</S.Name>
            <S.Description>저는 5년차 개발자입니다. 주로 react 개발을 위주로 하는 중 입니다.</S.Description>
          </S.ProfileDescription>
        </S.Profile>
        
        <S.MentorSkills>
          <S.Mentor>멘토</S.Mentor>
          <S.Skills>Skills</S.Skills>
        </S.MentorSkills>
      </S.ProfileWrapper>
      
      <S.ProfileBtn>All activity</S.ProfileBtn>
      <S.ProfileBtn>Article</S.ProfileBtn>
      <S.ProfileBtn>Comment</S.ProfileBtn>
    </>
```

Reply:
네 이 부분은 저도 좋은 것 같네요. 코드를 한 줄이라도 줄이려고 모아놓았네요.